File: PRAK110-2210817210015-MuhammadIrfan.c
#include<stdio.h>
#include<math.h>
    int main(){
        int alas = 5;
        int tinggi = 12;
        int miring = sqrt(pow((alas), 2) + pow((tinggi), 2));
        printf("Diketahui :\n");
        printf("Alas = %d cm\n", alas);
        printf("Tinggi = %d cm\n\n", tinggi);
        printf("Jawab :\n");
        printf("Sisi A = %d cm\n", tinggi);
        printf("Sisi B = %d cm\n", miring);
        printf("Sisi C = %d cm\n", alas);
        printf("Keliling = %d cm\n", alas+tinggi+miring);
        printf("Luas = %d cm", (alas*tinggi)/2);
        return 0;
    }

File: PRAK110-2210817210015-MuhammadIrfan.py
import math
alas, tinggi = 5, 12
miring = math.sqrt((alas ** 2) + (tinggi ** 2))
print("Diketahui :")
print("Alas = {} cm" .format(alas))
print("Tinggi = {} cm\n" .format(tinggi))
print("Jawab :")
print("Sisi A = {} cm" .format(tinggi))
print("Sisi B = {} cm" .format(round(miring)))
print("Sisi C = {} cm" .format(alas))
print("Keliling = {} cm" .format(round(alas+tinggi+miring)))
print("Luas = {} cm" .format(round((alas*tinggi)/2)))